src/server/data_store/data_store.h

```diff
@@ -0,0 +1,15 @@
+#ifndef DATA_STORE_H
+#define DATA_STORE_H
+
+#include <string>
+
+// TODO(joejoevictor): We should probably put this into a namespace
+class DataStore {
+public:
+  virtual ~DataStore() {}
+  // TODO(joejoevictor): Use optional as returned result
+  virtual ::std::string Get(::std::string key) = 0; // Pure virtual
+  virtual void Set(::std::string key, ::std::string value) = 0; // Pure virtual
```

Comment:
nit: same comments as value being a string-only. But I guess for now it's ok..? :/